authors/apps/authentication/tests/base_class.py

```diff
@@ -64,3 +70,10 @@ def create_another_user_in_db(self):
             username='roger',
             email='roger@mail.com',
             password='ia83naJS')
+
+    def create_article(self, user):
```

Comment:
Bring article creation into the `authentication` model mixes concerns. Perhaps a better place to put it would be in a method in your own test class?

Reply:
I think we had talked about this yesterday, about having a uniform base_test file. This is why I added it here. Oma is actually using the same one. So do @Karuhanga  think I should create another base class in tests folder for articles.

Reply:
@karuhanga, if you could please point out where it updated the previous migrations file. that would be nice. From my point of view I see the second created migration file and the initial one that Fahd had created. Note 0001__initial and 0002.

Reply:
Thanks for the catch on the constraint of 1-5. I will be implementing that and writing tests for that

Reply:
> I think we had talked about this yesterday, about having a uniform base_test file. This is why I added it here. Oma is actually using the same one. So do @Karuhanga think I should create another base class in tests folder for articles.

Yes, I think so, unless perhaps you fell that this will be better addressed when the base class is pulled out of the authentication app.

Reply:
> @Karuhanga, if you could please point out where it updated the previous migrations file. that would be nice. From my point of view I see the second created migration file and the initial one that Fahd had created. Note 0001__initial and 0002.

This was appropriately resolved, thank you.